src/main/java/org/reactome/addlinks/fileprocessors/HPAFileProcessor.java

```diff
@@ -0,0 +1,77 @@
+package org.reactome.addlinks.fileprocessors;
+
+import org.reactome.addlinks.EnsemblBiomartUtil;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class HPAFileProcessor extends FileProcessor<List<String>>
+{
+
+    public HPAFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public HPAFileProcessor()
+    {
+        super();
+    }
+
+    private static final int geneNameIndex = 0;
+    private static final int ensemblIdentifierIndex = 2;
+    private static final int uniprotIdentifierIndex = 4;
+
+    /**
+     * Build map of UniProt identifiers to Human Protein Atlas (HPA) identifiers that is used to create HPA cross-references in database.
+     * @return - Map<String, List<String>
+     */
+    @Override
+    public Map<String, List<String>> getIdMappingsFromFile()
+    {
+        Map<String, List<String>> mappings = new HashMap<>();
+        String dirToHPAFiles = null;
```

Comment:
Whenever possible, it is best to avoid initializing a value to `null`.  Using an empty value like `""` is preferable to reduce running into NullPointerExceptions.

Reply:
Done